diem-move/diem-framework/experimental/demo-cli/src/main.rs

```diff
@@ -51,13 +53,10 @@ async fn main() -> Result<()> {
     let json_rpc_url = args.jsonrpc_endpoint;
     println!("Connecting to {}...", json_rpc_url);
 
-    let client = BlockingClient::new(json_rpc_url);
+    let client = BlockingClient::new(json_rpc_url.clone());
 
-    let seq_num = client
-        .get_account(address)?
-        .into_inner()
-        .unwrap()
-        .sequence_number;
+    let seq_num =
+        request_utils::get_account_seq_num(json_rpc_url.as_str(), args.account_address).await?;
```

Comment:
This feels really weird to create a client, and then get the sequence number via a util that bypasses the client.  I'd flip these.

I'd also rename request_utils to rest_request_utils